docs/source/guides/staking_guide.rst

```diff
@@ -0,0 +1,149 @@
+=======================
+NuCypher Staking Guide
+=======================
+
+**Ursula Staking Actions**
+
++-------------------+-------------------------------------------------------------------------+
+| Action            |  Description                                                            |
++===================+=========================================================================+
+|  `stake`          | Initialize or view NuCpher stakes (used with `--value` and `--duration`)|
++-------------------+-------------------------------------------------------------------------+
+|  `divide-stake`   | Divide stake, used with `--value` and `--duration`                      |
++-------------------+-------------------------------------------------------------------------+
+|  `collect-reward` | Collect staking reward (Policy reward collection in future PR)          |
++-------------------+-------------------------------------------------------------------------+
+
+
+**Ursula Staking Options**
+
++----------------+----------------------------------------+
+| Option         |  Description                           |
++================+========================================+
+|  `--value`     | Stake value                            |
++----------------+----------------------------------------+
+|  `--duration`  | Stake duration of extension            |
++----------------+----------------------------------------+
+|  `--index`     | Stake index                            |
++----------------+----------------------------------------+
+|  `--list`      | List stakes (used with `stake` action) |
++----------------+----------------------------------------+
+
+
+
+Interactive Method
+------------------
+
+.. code:: bash
+
+    (nucypher)$ nucypher ursula stake
+
+
+*Initial a new stake*
+
+.. code:: bash
+
+    Stage a new stake? [y/N]: y
+
+    Current balance: 10000
+    Enter stake value in NU [1500]: 3000
+
+    Minimum duration: 30 | Maximum Duration: 365
+    Enter stake duration in periods (1 Period = 24 Hours): 90
+
+    ============================== STAGED STAKE ==============================
+
+    (Ursula)⇀Sienna Scales GreenYellow Ferry↽ (0x058D5F4cC9d52403c2F6944eC1c821a0e6E78657)
+    ~ Value      -> 3000 NU (30000000000000000000000 NU-wei)
```

Comment:
Just a silly comment: We need our own NU denominations (yay! NuCypher's weekly naming discussion!)

* "I'm staking _20 kman_" 
* "Oh no, I got slashed _100 tuxxy_!"
* ...

Reply:
At the very least, we can write `3e21 NU-wei` (idk if everyone will be familiar with that notation, but perhaps better than counting zeros) (crap, there are 22 zeros here, what's happening!)

Reply:
Yeah, bad conversion somewhere. I'll check for it.

Reply:
Actually, this looks correct

Reply:
:heavy_check_mark: (also following up in #830 )